lib/rubyscraper/scrapes.json

```diff
@@ -158,5 +158,201 @@
+      "fields":[
+        {
+          "field":"company",
+          "method":"find",
+          "path":"div.company"
+        },
+        {
+          "field":"position",
+          "method":"find",
+          "path":"div.title a"
+        },
+        {
+          "field":"url",
+          "method":"find",
+          "path":"div.title a",
+          "attr":"href"
+        },
+        {
+          "field":"location",
+          "method":"find",
+          "path":"div.location"
+        },
+        {
+          "field":"posting_date",
+          "method":"find",
+          "path":"div.date"
+        }
+      ]
+    },
+    "sub_page":{
+      "fields":[
+        {
+          "field":"description",
+          "method":"all",
+          "path":"div.job-listing-body p",
+          "loop_collect":"text",
+          "join":"\n"
+        }
+      ]
+    }
+  },
+  {
+    "name":"railsjobbers",
+    "base_url":"http://railsjobbers.com",
+    "summary":{
+      "url":"",
+      "has_sub_pages":"true",
+      "paginated":"false",
+      "loop":"div.page-content div#job-",
+      "fields":[
+        {
+          "field":"company",
+          "method":"find",
+          "path":"div.post-meta span[title='employer']"
+        },
+        {
+          "field":"position",
+          "method":"find",
+          "path":"div.post-title a"
+        },
+        {
+          "field":"url",
+          "method":"find",
+          "path":"div.post-title a",
+          "attr":"href"
+        },
+        {
+          "field":"posting_date",
+          "method":"find",
+          "path":"a.date",
+          "attr":"title"
+        }
+      ]
+    },
+    "sub_page":{
+      "fields":[
+        {
+          "field":"location",
+          "method":"first",
+          "path":"div.field-item div.bold"
+        },
+        {
+          "field":"description",
+          "method":"all",
+          "path":"div#job_detail_description p",
+          "loop_collect":"text",
+          "join":"\n"
+        }
+      ]
+    }
+  },
+  {
+    "name":"toprubyjobs",
+    "base_url":"https://toprubyjobs.com",
+    "summary":{
+      "url":"",
+      "has_sub_pages":"true",
+      "paginated":"false",
+      "loop":"ul#jobs li.job",
+      "fields":[
+        {
+          "field":"position",
+          "method":"find",
+          "path":"h4.job-title a"
+        },
+        {
+          "field":"url",
+          "method":"find",
+          "path":"h4.job-title a",
+          "attr":"href"
+        }
+      ]
+    },
+    "sub_page":{
+      "fields":[
+        {
+          "field":"company",
+          "method":"find",
+          "path":"dd:nth-child(2) a"
+        },
+        {
+          "field":"location",
+          "method":"first",
+          "path":"dd:nth-child(4)"
+        },
+        {
+          "field":"description",
+          "method":"all",
+          "path":"div.description p",
+          "loop_collect":"text",
+          "join":"\n"
+        }
+      ]
+    }
```

Comment:
This is getting long...maybe there is a way to break this out with one file for each company? Or have a hash for each company, and then loop through all the companies?